paasta_tools/cli/utils.py

```diff
@@ -742,3 +744,34 @@ def pick_slave_from_status(status, host=None):
     else:
         slaves = status.marathon.slaves
         return slaves[0]
+
+
+def get_task_from_instance(cluster, service, instance, slave_hostname=None, task_id=None):
```

Comment:
Do you know where about I could get the same information? Also doesn't `paasta status` just print strings?